src/configuration/ConfigurationPage.tsx

```diff
@@ -92,20 +100,38 @@ const useStyles = makeStyles(
 export interface ConfigurationPageProps {
   menu: MenuSection[];
   user: User;
+  versionInfo: VersionInfo;
   onSectionClick: (sectionName: string) => void;
 }
 
 export const ConfigurationPage: React.FC<ConfigurationPageProps> = props => {
-  const { menu: menus, user, onSectionClick } = props;
+  const {
+    menu: menus,
+    user,
+    onSectionClick,
+    versionInfo: { dashboardVersion, coreVersion }
+  } = props;
   const classes = useStyles(props);
+  const theme = useTheme();
+  const isSmUp = useMediaQuery(theme.breakpoints.up("sm"));
+
+  const VersionInfoComponent = () => (
+    <VersionInfo
+      dashboardVersion={dashboardVersion}
+      coreVersion={coreVersion}
+    />
+  );
```

Comment:
Defining components inside another component is an anti-pattern. It will cause the component to remount on every render, which would make its internal state obsolete. In this scenario it's best to define it as a variable:
```suggestion
  const renderVersionInfo = (
    <VersionInfo
      dashboardVersion={dashboardVersion}
      coreVersion={coreVersion}
    />
  );
```